microsoft-edge/devtools-guide-chromium/device-mode/testing-other-browsers.md

```diff
@@ -118,30 +111,12 @@ When you are done, learn how to work with the simulator through [Apple Developer
 > To avoid having to open Xcode every time you want to use the iOS Simulator, open <!--Xcode, or iOS Simulator?-->it, right-click the iOS Simulator icon in your dock, and then select **Keep in Dock**.  Now just click the icon whenever you need it.
 
 
-<!-- ------------------------------ -->
-#### Microsoft Edge (EdgeHTML)
-
-If you need to test your website or app with Microsoft browsers and don't have the necessary versions of Windows to do so, you can use BrowserStack, which supports testing of many combinations of Microsoft browsers and operating systems both past and present.  For example, you can test all versions of Microsoft Edge (Chromium) from version 80 onwards, and Microsoft Edge (EdgeHTML) versions 15 through 18.  Testing of Microsoft Edge is free on BrowserStack. <!-- For more information, see [Microsoft Edge Browser Testing](https://www.browserstack.com/test-on-microsoft-edge-browser) at BrowserStack.  todo: update 404 link, article title not found at site -->
-
-
 <!-- ====================================================================== -->
 ## Cloud-based emulators and simulators
 
 If you aren't able to use the emulators and you don't have access to real devices, then cloud-based emulators are the next-best thing.  A big advantage of cloud-based emulators over real devices and local emulators is that you can automate unit tests for your site across different platforms.
```

Comment:
```suggestion
If Device Emulation in Microsoft Edge DevTools isn't sufficient for your testing needs, you can test using real devices instead. If you don't have access to the real devices you want to test on, then setting up a virtual machine, or using a cloud-based testing service can help. A big advantage of cloud-based testing services is that they can help you automate testing for your site across different platforms and form factors.
```

Reply:
result:
```
If Device Emulation in Microsoft Edge DevTools isn't sufficient for 
your testing needs, you can test by using real devices instead.  If 
you don't have access to the real devices that you want to test on, 
setting up a virtual machine or using a cloud-based testing service 
can help.  A big advantage of cloud-based testing services is that 
they can help you automate testing for your site across different 
platforms and form factors.
```

Reply:
that change wasn't committed.  went w/ later draft from CB, which avoids negative lead-in (inherited from upstream) focused on different scenario:
Cloud-based emulators and simulators are commercial solutions to test your website on different devices and browsers without installing anything on your development machine.  They are useful when you don't have access to real devices or when you can't use local emulators. Cloud-based solutions are also useful for automating tests across different platforms.

@bwalderman